javalib/src/main/scala/java/time/Month.scala

```diff
@@ -0,0 +1,103 @@
+package java.time
+
+import java.time.temporal._
+
+final class Month private(name: String,
+                          ordinal: Int,
+                          val minLength: Int,
+                          val maxLength: Int)
```

Comment:
Should only keep one field for the length (`minLength ` as `defaultLength`). You put compute the `maxLength` when you need it.